src/m_misc.c

```diff
@@ -414,6 +415,13 @@ default_t defaults[] = {
     "OpenAL resampler (\"nearest\", \"linear\" (default), \"cubic\")"
   },
 
+  {
+    "snd_hrtf",
+    (config_t *) &snd_hrtf, NULL,
+    {-1}, {-1, 1}, 0, ss_gen, wad_no,
```

Comment:
disable by default?

Reply:
It is "auto" by default in GZDoom and "enabled" in OpenAL Soft (which is problematic).

Reply:
So this takes priority over any `alsoft.ini` on the system right? I believe OpenAL Soft looks in both the user folder and game folder for this file. It would be nice to see the OpenAL Soft log file to know why this was happening for one person but not others.

Environment variables:
`ALSOFT_LOGFILE` set to anything (e.g. `C:\Woof\alsoft_log.txt`)
`ALSOFT_LOGLEVEL` set to `3` (most verbose)

Reply:
We don't even have the most remote support for positional audio, so IMHO "disabled" is the only reasonable default.

Reply:
Also, HRTF shouldn't even work if the .dll doesn't have the "Default HRTF" embedded which depends on how the OpenAL Soft dll was compiled. The log file would also explain this.

Reply:
I don't know, this is what GZDoom does and it works. See the discussion in the Woof thread: [[doomworld]](https://www.doomworld.com/forum/topic/112333-this-is-woof-1051-dec-28-2022-updated-winmbf/?do=findComment&comment=2630669).
Not sure, why we need a log? GZDoom doesn't log this I think.

For me, the default settings work fine with my headphones in both Woof and GZDoom (the HRTF mode definitely works here, it's even distracting).

Reply:
I'm not even sure if this should be s user choice at all.

Reply:
> I'm not even sure if this should be a user choice at all.

So should we just set it to `ALC_DONT_CARE_SOFT`? I think we should have the `snd_hrtf` option since we are also overwriting the `alsoft.ini` config. Maybe someone will need to force disable HRTF.

Reply:
I have understood that @liPillON's reduced sound quality was because HRTF was accidently enabled on his system?

Reply:
> I think `ALC_STEREO_BASIC_SOFT` is what you want for controlling this directly from Woof. There shouldn't be a setting in woof.cfg at all.

Here is qoute from pantheon (@ak-988) on Discord: "Weird but setting stereo mode to headphones sounds really bad, and I use headphones" https://discord.com/channels/435469554173476866/1051935485540450386/1094026578503143484

I think `snd_hrtf` is not a problem, copying from GZDoom is just safe.

Reply:
Will we need another config key for different stereo profiles?

Reply:
> Here is qoute from pantheon (@ak-988) on Discord: "Weird but setting stereo mode to headphones sounds really bad, and I use headphones" https://discord.com/channels/435469554173476866/1051935485540450386/1094026578503143484

What happened there is OpenAL Soft automatically turned on HRTF when `stereo-mode` was changed to `headphones`. So yes, it's going to sound bad since it's trying to apply binaural audio to Woof's simple panning. See:

```
alsoft.ini:
[general]
stereo-mode = speakers (or auto if not using a USB headset)

C:\Games\Doom\Woof>openal-info64.exe
Device output mode: Stereo (basic)
```

```
alsoft.ini:
[general]
stereo-mode = headphones

C:\Games\Doom\Woof>openal-info64.exe
Device output mode: Stereo (HRTF)
```


> I think snd_hrtf is not a problem, copying from GZDoom is just safe.

If "Auto" works, I think you are right.

Reply:
GZDoom has: `snd_alresampler`, `snd_aldevice` and `snd_hrtf`, no stereo profiles.
The reason I want to copy the GZDoom settings is because the author of OpenAL Soft implemented them and I'm sure OpenAL Soft will work in GZDoom.

Reply:
But `snd_hrtf 1` sounds good with my headphones, definitely better than liPillON recordings. Perhaps there are some bugs in the OpenAL or Windows drivers, so it's wise not to impose any defaults and have a config option. We have many config options, it's not a big problem 😄

Reply:
Here's a simple test. Start with your branch and the "Auto" setting:
```
woof.cfg:
snd_hrtf -1
```
Let's simulate a USB headset (like what liPillON is using):
```
alsoft.ini:
[general]
stereo-mode = headphones
```
Launch the game. It sounds bad. Let's check OpenAL Soft's log:
```
alsoft.log:
[ALSOFT] (II) Adding built-in entry "!1_Built-In HRTF"
[ALSOFT] (II) Key default-hrtf not found
[ALSOFT] (II) Loading !1_Built-In HRTF...
[ALSOFT] (II) Detected data set format v3
[ALSOFT] (II) Loaded HRTF Built-In HRTF for sample rate 48000hz, 64-sample filter
[ALSOFT] (II) Key hrtf-size not found
[ALSOFT] (II) Key hrtf-mode not found
[ALSOFT] (II) 1st order + Full HRTF rendering enabled, using "Built-In HRTF"

```
Okay, so now we know that "snd_hrtf -1" isn't going to solve the problem.

Reply:
> So, are you suggesting to just disable HRTF and remove the snd_hrtf config option?

That's right. I asked liPillON so share a log from your build to confirm 100%. But I will leave it to you if you want to wait or not.

Reply:
Also confirmed by liPillON: [[doomworld]](https://www.doomworld.com/forum/topic/112333-this-is-woof-1051-dec-28-2022-updated-winmbf/?page=61&tab=comments#comment-2630872)